lib/registries/package.json

```diff
@@ -28,6 +29,7 @@
     "ember-i18n": "*",
     "ember-i18n-inject": "*",
     "ember-moment": "*",
+    "ember-on-modifier": "*",
     "ember-page-title": "*",
```

Comment:
`@ember/render-modifers` and `ember-on-modifiers` are needed for `OsfDialog` to work in registries.

Reply:
Ah sorry, #742 is still in QA